homework.md

```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the Web browsers to create a dynamic and interactive experience
```

Comment:
### Side Note here that you have to take into consideration:

- Nowadays, JavaScript is an all-purpose programming language—meaning it runs across the entire software stack. The most popular application of JavaScript is on the client side (aka frontend), but since Node.js came on the scene, many people run JavaScript on the server side (aka backend) as well not forgetting that when React Native came along we can now write Mobile Applications using JavaScript finally with Electron also we can build Desktop Applications using JavaScript.

### Key to this question is not answer "what JavaScript does" cause that is a question on it's own but is to answer "what is JavaScript", e.g Object Oriented Programming Lang or Interpreted Programming Lang or Compiled etc...

Reply:
Thank you Ntshembo